types/react-virtualized/react-virtualized-tests.tsx

```diff
@@ -196,6 +196,7 @@ export class ArrowKeyStepperESMExample extends PureComponent<any, any> {
 import { List } from 'react-virtualized';
 
 export class AutoSizerExample extends PureComponent<any, any> {
+    context: any;
```

Comment:
PureComponent inherits from Component which already declares `context: any` as property of the class. So adding it here doesn't bring a benefit

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/43567b6a47244cd672f372c1259c8e9132afa675/types/react/index.d.ts#L479

Reply:
@kaoDev It does now but, as explained in the PR description, we'll move to `context: unknown` in `React.Component`.

Reply:
oh sorry, I should read more PR descriptions 🤦